doc/releasenotes/15_0_0_summary.md

```diff
@@ -54,6 +54,34 @@ Please see the VDiff2 [documentation](https://vitess.io/docs/15.0/reference/vrep
 
 ### New command line flags and behavior
 
+#### Support for additional compressor and decompressor during backup & restore
+We have introduced feature to use your own compressor & decompressor during backup & restore, instead of relying on default compressor. 
```

Comment:
```suggestion
Backup/Restore now allow you many more options for compression and decompression instead of relying on the default compressor (pgzip). 
```